tools/runners/run-test-suite-test262.sh

```diff
@@ -14,108 +14,105 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-TARGET=$1
-shift
-
-PARSE_ONLY=""
-VERBOSE_REPORT=0
-TIMEOUT=5
-
-function usage() {
-  echo "Usage: $0 {path to engine} [-p] [-v] [-t {timeout in seconds}]" >&2
-  echo "  -p - parser only mode" >&2
-  echo "  -v - verbose report mode" >&2
-  echo "  -t - timeout for each test" >&2
+ENGINE="$1"
+PATH_TO_TEST262="$2"
+REPORT_PREFIX="report"
+RUN_PIDS=""
+RESULT_OK=1
+TIMEOUT="3s"
 
+if [ $# -lt 2 ]
+then
+  echo "This script performs parallel test262 compliance testing of the specified engine."
+  echo ""
+  echo "Usage:"
+  echo "  1st parameter: JavaScript engine to be tested."
+  echo "  2nd parameter: path to the directory with official test262 testsuite."
+  echo "  3rd parameter: (optional) call this script with the '--sub-chapters' flag to print the detailed report."
+  echo ""
+  echo "Example:"
+  echo "  ./run-test-suite-test262.sh <engine> <test262_dir> --detail"
```

Comment:
Maybe copy&paste error.  I think `--detail` is not a valid option. Change to `--sub-chapters`